packages/flutter/test/widgets/sliver_cross_axis_group_test.dart

```diff
@@ -0,0 +1,188 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
For each, there should be a vertical, vertical/reverse, horizontal, and horizontal/reverse